litmus-portal/graphql-server/pkg/database/mongodb/operations/project.go

```diff
@@ -98,3 +111,32 @@ func UpdateInvite(ctx context.Context, projectID, userName string, invitation db
 	}
 	return nil
 }
+
+//CancelInvite ...
+func CancelInvite(ctx context.Context, projectID string, userName string) error {
```

Comment:
This function is same as `removeMember`. Let's have one function only for the same and give it a generalized name like `removeInvitation`